rinja/src/filters/escape.rs

```diff
@@ -309,6 +318,20 @@ const _: () = {
         NeedsEscaping(&'a T, E),
     }
 
+    impl<T, E> FastWritable for Wrapped<'_, T, E>
+    where
+        T: AsRef<str> + fmt::Display + ?Sized,
```

Comment:
I think the `fmt::Display` bound can be removed since we only ever use `as_ref()`.

Reply:
Removed.